index.js

```diff
@@ -199,7 +200,12 @@ exports.createDiff = function saveDiff(opts, callback) {
             };
 
         buildDiffImage(result.first, result.second, diffOptions, function(result) {
-            result.save(opts.diff, callback);
+            if (save) {
```

Comment:
You can use `opts.save` directly here without having to do [this](https://github.com/flore77/looks-same/blob/125c432f5f58e65eb9d68540c952dc596ba132c4/index.js#L191).